src/styles/components/_list.scss

```diff
@@ -0,0 +1,32 @@
+$node-inset: 1rem;  // approximate scaled margin + 'outer-trim' of a node
```

Comment:
Can we replace this with a CSS variable to reduce our dependence on SASS?

Reply:
Will remove entirely, it isn't used here and is just borrowed from node-list.scss